### ISSUE TITLE
How do you get the training time on each epoch using TPUEstimator?

### ISSUE THREAD

**Issue by elvinjgalarza** [google-research/bert#907]
I am able to see INFO:tensorflow:loss = 134.62343, step = 97

but not the time.
